README.md

```diff
@@ -1,3 +1,5 @@
+# <img src="https://raw.githubusercontent.com/open-atmos/PySDM/main/.github/pysdm_logo.svg" style="height:50pt" alt="pysdm logo">
```

Comment:
the png logo is 100x146, hence this will result in blurry image on PyPI, let's change to `width=100 height=146`?

Reply:
addressed in https://github.com/open-atmos/PySDM/pull/1381/commits/66e08d613ee8aedade0f738f07fba8c6dfec9b16